lib/collections.js

```diff
@@ -46,8 +72,27 @@ var longAndShort = {};
 longAndShort['longest'] = '';
 longAndShort['shortest'] = paragraph;
 
+// Look at the uniqueWords array. Find the longest word and store that word as the value of the 'longest' key. Find the shortest word and store that word as the value of the 'shortest' key.
+//
 /* your code starts here */
 
+// ATTEMPT #1 (doesn't work):
+ var length = 0;
+ var sortedWords = uniqueWords.sort();
+
+for (var i = 0; i < sortedWords.length; i++) {
+  length = sortedWords[i].length;
+  if ((length - 1) === false) {
```

Comment:
What you really need to be doing here is comparing each word to both the longest word so far and the shortest word so far. If it's longer than the longest word, it becomes the new longest word so far; if it's shorter than the shortest word, it becomes the new shortest word so far.